cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -74,6 +89,31 @@ func newCreateManagedJobCmd() *cobra.Command {
 	return cmd
 }
 
+var isManagementCluster = func () (bool, error) {
+	//  check for a label with a key of ext-hypershift.openshift.io/cluster-type and a value of management-cluster
+	connection, err := oocm.NewConnection().Build()
+	if err != nil {
+		return false, fmt.Errorf("failed to create OCM connection: %v", err)
+	}
+	defer connection.Close()
+
+	labelsListResponse, err := connection.ClustersMgmt().V1().Clusters().Cluster(options.clusterID).ExternalConfiguration().Labels().List().Send()
+	if err != nil {
+		if labelsListResponse.Status() == 400 {
+			return false, nil
+		}
+		return false, fmt.Errorf("failed to get label from ocm: %v", err)
+	}
+
+	for _, label := range labelsListResponse.Items().Slice() {
```

Comment:
Do we need to check the length of the slice before iterating?

Reply:
What is the  idiomatic way to do it in go? If the length of the slice is 0, there will be 0 iterations of the for loop so what is the additional need to check for len()?